snuba/replacer.py

```diff
@@ -90,10 +89,12 @@ def get_projects_query_flags(project_ids):
 
 
 class ReplacerWorker(AbstractBatchWorker):
-    def __init__(self, clickhouse, dist_table_name, metrics=None):
+    def __init__(self, clickhouse, dataset, metrics=None):
```

Comment:
The replacer makes sense only on events. Once we are done with the introduction of datasets we can revisit this abstraction to make it strictly tied to the events dataset. For the time being I am passing the dataset through.

Reply:
Makes sense.